### DISCUSSION TITLE
Week 3 - Knowing Linux Basics

### DISCUSSION THREAD

**Discussion by MichaelCade** [MichaelCade/90DaysOfDevOps#27] [General]
https://github.com/MichaelCade/90DaysOfDevOps#learning-a-programming-language

**Comment by adtyap26** [comment]
On the days15.md about Linux commands for devops, you've mentioned the use of cat and grep at the same time with piping. I don't think it's really necessary. Instead of using, for example, cat days15 | grep "90DaysOfDevOps," which results in "Hello #90DaysOfDevOps," you can use only grep without cat, like this: grep "90DaysOfDevOps" days15, which also gives the same result but with less command that we have to write.

I am enjoying your content so much, Cherss.

**Reply by jkilungya** [reply]
I will try this today grep "90DaysOfDevOps" days15 to see if it works
